packages/cli/README.md

```diff
@@ -127,3 +127,615 @@ jobs:
       - run: npm ci
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
+
+## Custom Plugins
+
+One of the main features of Code PushUp is the ability to write custom plugins to track your own metrics.
+It enables you to implement neraly any kind of metric you want to track with minimum effort.
+In this section we will go through the steps to create a custom plugin and integrate it in your project.
+
+### Set up
+
+To start crafting custom plugins you need a minimum `code-pushup.config.ts` file including the `persist` and `plugins`
```

Comment:
```suggestion
To start crafting custom plugins you need a minimum `code-pushup.config.(ts|js|mjs)` file including the `persist` and `plugins`
```
We support `ts`, `mjs` and `js`, I would either omit the file format or include all options and emphasise the example is for `ts`.